apps/emqx_authn/test/emqx_authn_api_SUITE.erl

```diff
@@ -247,6 +247,15 @@ test_authenticator(PathPrefix) ->
         <<"connected">>,
         LookFun([<<"status">>])
     ),
+
+    ?assertMatch(
+        {ok, 404, _},
+        request(
+            get,
+            uri(PathPrefix ++ [?CONF_NS, "unknown_auth_chain", "status"])
```

Comment:
Nit: maybe also add a test for an unknown authenticator status under a listener?

Reply:
That's implicit since `PathPrefix` is a param to the test and it is also called as `test_authenticator(["listeners", ?TCP_DEFAULT])`.
If that's what you mean.

Reply:
Aha, I see.  That's what I meant.  :smiley_cat: